tests/unit/TestStationDAO.unitTest.ts

```diff
@@ -16,16 +18,23 @@ describe("Preparers DAO", () => {
         it("returns data on successful query", async () => {
             mockDocumentClientWithReturn("scan", "success");
             const dao = new TestStationDAO();
-            const output = await dao.getAll();
+            const output = await dao.getAll(TEST_STATION_STATUS.ACTIVE);
             expect(output).to.equal("success");
```

Comment:
can you please move "success" to enum... guess it had already been done

Reply:
Requested change completed.